obelisk/cpp/zoo/robot/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.5)
+project(dynamixel_interface LANGUAGES CXX)
+
+message(STATUS "Configuring Dynamixel Interface")
+
+# Find the Dynamixel SDK library
+find_package(ament_cmake REQUIRED)
+find_package(dynamixel_sdk REQUIRED)
+
+include_directories("$ENV{OBELISK_ROOT}/.pixi/envs/dev/include/dynamixel_sdk")
+include_directories("$ENV{OBELISK_ROOT}/.pixi/envs/dev-no-gpu/include/dynamixel_sdk")
```

Comment:
i feel like this isn't the best way to do this include. the end user may not even be using pixi, so this would fail for them. is there a better way to specify this dep?

Reply:
also, just for future reference, if we do end up doing hacks like this in the future, we shouldn't hardcode env names in these paths, since those could potentially change and break things. There's a `PIXI_ENVIRONMENT_NAME` environment variable after pixi shells are activated that you can instead reference.

Reply:
Yes, using `<package>_INCLUDE_DIRS` solved the issue.